src/content/nft/index.md

```diff
@@ -316,56 +315,16 @@ When you mint an NFT, a few things have to happen:
 - The transactions that confirm the above need to be added to a block and "immortalised" on the chain.
 - The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximise their earnings.
 
-All these tasks are done by miners. And they let the rest of the network know about your NFT and who owns it. This means mining needs to be sufficiently difficult, otherwise anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership. There are lots of incentives in place to make sure miners are acting honestly.
+All these tasks are done by block producers and validators. Block proposers add your NFT transaction to a block and broadcast it to the rest of the network. Validators check that the transaction is valid and then add it to their databases. There are lots of crypto-economic incentives in place to make sure validators are acting honestly. Otherwise, anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership.
 
-[More on mining](/developers/docs/consensus-mechanisms/pow/)
+#### NFT security {#nft-security}
 
-#### Securing your NFT with mining {#securing-nfts}
+Ethereum's security comes from its proof-of-stake consensus mechanism - validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ether to change it, and even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```

Comment:
```suggestion
Ethereum's security comes from its consensus mechanism which uses proof-of-stake. Validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it, and even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```

Also:

> even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. 

Just noting this as it could potentially cause confusion... do we want to mention this here?

Reply:
yea i suppose it is a bit too nuanced a discussion for this page. 
```suggestion
Ethereum's security comes from its proof-of-stake consensus mechanism - validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ether to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```